templates/service-bus-sessions.json

```diff
@@ -101,6 +101,31 @@
         },
         "Try": {
           "actions": {
+            "Abandon_the_message_in_a_queue_2": {
+              "description": "If something goes wrong processing the message, we must abandon it to release it for pickup by another run",
+              "inputs": {
+                "host": {
+                  "connection": {
+                    "name": "@parameters('$connections')['servicebus']['connectionId']"
+                  }
+                },
+                "method": "post",
+                "path": "/@{encodeURIComponent('')}/messages/abandon",
+                "queries": {
+                  "lockToken": "@triggerBody()?['LockToken']",
+                  "queueType": "Main",
+                  "sessionId": "@triggerBody()?['SessionId']"
+                }
+              },
+              "runAfter": {
+                "Send_initial_message_to_topic": [
+                  "Failed",
+                  "Skipped",
+                  "TimedOut"
+                ]
+              },
+              "type": "ApiConnection"
+            },
```

Comment:
Will this ever execute? If the trigger doesn't succeed then it won't generate a run, correct?

Reply:
The 'send initial message to topic' is the placeholder in the template for whatever logic the user wants to do w/ the message that was put on SB, not the trigger that kicked off the run. so yes, if that action fails the abandon must & would execute to abandon the message that triggered the entire workflow.

Reply:
Never mind, for some reason I thought it was running right after the trigger, but I see now that it's running after the 'do some work' step. This makes sense.
But it would be helpful if this was called something like "Abandon initial message from the queue" or something similar instead of "Abandon the message in a queue 2" as when users read this logic app they won't see the buried "Abandon the message in a queue" and may assume it's a different queue (since you have "queue 2" in the name).